src/interfaces/resources/resourcesUI.py

```diff
@@ -0,0 +1,46 @@
+"""
+
+    This file provides a pure GUI interface for resources.
+    This component is overall UI manager for resources tab.
+
+"""
+
+from PyQt5.QtCore import pyqtSignal
+
+from ..widgets import StackLayout, Frame
+
+
+class ResourcesUI(Frame):
+
+    _stack = None
+    _to_controller_signal = pyqtSignal()
+    _to_add_view_signal = pyqtSignal()
+
+    def __init__(self, *args, **kwargs):
+        super(ResourcesUI, self).__init__(*args, **kwargs)
+
+        # create layout for interface
+        self._stack = StackLayout(self)
```

Comment:
I get an error when running this code because this `StackLayout` call doesn't define margin parameters that are passed down to the `Layout` class.

Reply:
fixed